### PR TITLE
[CST] [DDL] [Backend] Added 5103 letters to DDL

### PR DESCRIPTION
**Pull request by samcoforma**
## Summary

- *This work is behind a feature toggle (flipper): YES
- Created `cst_include_ddl_5103_letters` feature toggle
- Updated `app/controllers/v0/claim_letters_controller.rb` to include `5130` doctypes when `cst_include_ddl_5103_letters` is enabled
- Updated `spec/controllers/v0/claim_letters_controller_spec.rb` unit test

## Related issue(s)

- [Add 5103 letter(s) to DDL #72464](https://github.com/department-of-veterans-affairs/va.gov-team/issues/72464)

## Testing done

- [x] *New code is covered by unit tests*

## What areas of the site does it impact?
CST DDL

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

File: app/controllers/v0/claim_letters_controller.rb [hunk 1]
```diff
@@ -30,9 +30,13 @@ def service
     # 27: Board Of Appeals Decision Letter
     # 184: Notification Letter (e.g. VA 20-8993, VA 21-0290, PCGL)
```
Comment (pmclaren19) [inline app/controllers/v0/claim_letters_controller.rb:31]: Can we remove line and 32? I dont see that letter anywhere here

File: spec/controllers/v0/claim_letters_controller_spec.rb [hunk 2]
```diff
@@ -39,8 +40,24 @@
     end
   end
 
-  describe '#index when "cst_include_ddl_boa_letters" feature flag is disabled' do
```
Comment (pmclaren19) [inline spec/controllers/v0/claim_letters_controller_spec.rb:42]: what happened to the tests from this describe?

File: spec/controllers/v0/claim_letters_controller_spec.rb [hunk 3]
```diff
@@ -39,8 +40,24 @@
...
+  describe '#index when "cst_include_ddl_5103_letters" feature flag is enabled' do
```
Comment (pmclaren19) [inline spec/controllers/v0/claim_letters_controller_spec.rb:43]: id update the describe to note that `cst_include_ddl_boa_letters` is disabled

**Review by pmclaren19 — APPROVED** [review]
LGTM!

**Review by pmclaren19 — APPROVED** [review]

**Review by rjohnson2011 — APPROVED** [review]

**Pull request merged**